src/samples/audio/smart_amp_test.c

```diff
@@ -692,6 +669,9 @@ static int smart_amp_copy(struct comp_dev *dev)
 
 		buffer_release(buf);
 	}
+#if CONFIG_IPC_MAJOR_4
+	sad->feedback_buf = NULL;
```

Comment:
so on every copy you have to re-assign the feedback buffer... TBH I'm not sure what smart-amp-test is actually for (it's "just" a test, right? So it isn't used in any real audio streaming?) but this seems rather expensive

Reply:
Yeah, this will need a comment to explain the specific case if needed other wise it could be copy and pasted into other samples.